mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDaoImpl.java

```diff
@@ -205,6 +206,19 @@ public Future<JobExecution> updateBlocking(String jobExecutionId, JobExecutionMu
     return promise.future();
   }
 
+  @Override
+  public Future<Boolean> deleteJobExecutionsByIds(List<String> ids, String tenantId) {
+    Promise<RowSet<Row>> promise = Promise.promise();
+    try {
+      String query = format(DELETE_BY_IDS_SQL, convertToPsqlStandard(tenantId), TABLE_NAME, ids.stream().collect(Collectors.joining("','")));
+      pgClientFactory.createInstance(tenantId).execute(query, promise);
+    } catch (Exception e) {
+      LOGGER.error("Error deleting jobExecution by ids {} : Exception : ", ids, e);
+      promise.fail(e);
+    }
+    return promise.future().map(updateResult -> updateResult.rowCount() == 1);
```

Comment:
if more than one job execution is deleted would the rowCount reflect that? 
if so the condition should be `updateResult.rowCount() > 0`.

Reply:
Hi @okolawole-ebsco - Updated changes as per review comment, Thanks 🙂